src/main/java/net/fabricmc/loom/task/RemapTaskConfiguration.java

```diff
@@ -139,7 +139,7 @@ private void trySetupSourceRemapping() {
 			}
 
 			sourcesJarTask.getArchiveClassifier().convention("dev-sources");
-			sourcesJarTask.getDestinationDirectory().set(new File(getProject().getBuildDir(), "devlibs"));
+			sourcesJarTask.getDestinationDirectory().set(new File(getProject().getLayout().getBuildDirectory().getAsFile().get(), "devlibs"));
```

Comment:
Sorry, I wasnt clear, this one also needs updating.

Reply:
Nope you were clear just had to fix the repo on my side since it was out of date